setupCacheEdits.ts

```diff
@@ -35,8 +37,10 @@ export function setupCacheEdits<B extends Bot>(
     if (guild) guild.memberCount--;
     // Update non-memory cache
     bot.cache.guilds.get(guildID).then((g) => {
-      g.memberCount++;
-      bot.cache.guilds.set(g);
+      if (g) {
+        g.memberCount++;
```

Comment:
Should this be --?

Reply:
I was only changing the type errors but yes, it should be.